extensions/github/src/open-pull-request-menu.tsx

```diff
@@ -26,26 +26,21 @@ function getMaxPullRequestsPreference(): number {
   return getBoundedPreferenceNumber({ name: "maxitems" });
 }
 
-function getCheckStateEmoji(pr: PullRequestFieldsFragment): string | null {
-  const checkState = pr.commits.nodes ? pr.commits.nodes[0]?.commit.statusCheckRollup?.state : null;
-  switch (checkState) {
+function getPullRequestStatusIcon(pr: PullRequestFieldsFragment): { source: Icon | string } {
+  const pullRequestStatus = pr.commits.nodes ? pr.commits.nodes[0]?.commit.statusCheckRollup?.state : null;
+  switch (pullRequestStatus) {
     case "SUCCESS":
-      return "✅";
+      return { source: Icon.Check };
     case "ERROR":
     case "FAILURE":
-      return "⚠️";
+      return { source: Icon.Xmark };
     case "PENDING":
-      return "⏱️";
+      return { source: Icon.Clock };
     default:
-      return null;
+      return { source: "pull-request.svg" };
   }
 }
```

Comment:
Are you sure the colors make it look better and easier to scan? I personally find that they make it look a bit messy:

<img width="621" alt="Screenshot 2024-02-26 at 07 47 44" src="https://github.com/raycast/extensions/assets/11052366/bb1aea26-28d1-4463-8d24-7dc114aac0ce">

<img width="621" alt="Screenshot 2024-02-26 at 07 50 01" src="https://github.com/raycast/extensions/assets/11052366/48e926a4-1e64-4b8d-b0e7-840811944fdc">

Colors in the menu bar hardly work well, it's no wonder that Apple rarely uses them. On the other hand, I'd agree it looks good In Raycast since the UI is more complex:

<img width="886" alt="Screenshot 2024-02-26 at 07 57 50" src="https://github.com/raycast/extensions/assets/11052366/4169a7f1-b35b-4535-8638-320076482ee6">

But that's just my opinion. Tell me your final word.

Reply:
I think it makes it look a lot better. I actually think Apple does it wrong by not using colors. But this is just my own personal opinion.

Reply:
It's impossible to maintain design consistency when using a bunch of colors in the menu bar. Colors can make the menu bar distracting, so you should only focus on it when necessary. Apple uses colors in the menu bar sparingly as shown in this example:

<img width="346" alt="Screenshot 2024-02-26 at 12 49 56" src="https://github.com/raycast/extensions/assets/11052366/4c96b723-6f35-4cb8-9c60-47dec0676cb9">

Reply:
Distraction is helpful when it focuses your attention where needed. For instance, the "Unread Notification" menu bar command turns orange, standing out from other icons and signaling a new GitHub notification.

In our case, you don't need to stress the UI by using colors because the icons are sufficient to differentiate between the statuses. By doing so, you create a visual hierarchy that directs your attention first to the icon and only then to the text, which is what matters in the end. It's no wonder that SF Symbols are so commonly used in menu bar apps. They are designed to integrate seamlessly with the San Francisco system font conveying information graphically without visual distractions.

Reply:
Maybe just make it a setting?

Reply:
I agree with @nesl247, as it seems to be more a personal preference thing, we might want to make a setting out of it.

Reply:
As you want, I'm not going to block the PR anyway. I'm fine shipping it without colors 🙂